devsite/devsite/test_settings.py

```diff
@@ -106,17 +103,20 @@ def root(*args):
 }
 
 # It expects them from the project's settings (django.conf.settings)
-WEBPACK_LOADER = {}
+WEBPACK_LOADER = {
+    'FIGURES_APP': {
+        'BUNDLE_DIR_NAME': 'figures/',
+        'STATS_FILE': 'tests/test-webpack-stats.json',
+    }
+}
 CELERYBEAT_SCHEDULE = {}
 FEATURES = {}
 
 # Declare values we need from server vars (e.g. lms.env.json)
 ENV_TOKENS = {
```

Comment:
Remove the commented code.

```suggestion
ENV_TOKENS = {}
```